pages/linux/wodim.md

```diff
@@ -0,0 +1,17 @@
+# wodim
+
+> Command for record/burning data using a optical drive to CD or DVD.
+> Caution when using this command.
+> There are many ways this command can be destructive, the following below is meant as a reminder of wodims syntax and not as the only source of documentation.
+
+- Display optical drives avaliable to wodim:
+
+`wodim --devices`
+
+- Burn file to disc in optical drive:
+
+`wodim -eject -tao speed=0 dev=/dev/{{optical_drive}} -v -data /{{file.iso}}`
+
+- Record a pure CD-DA audio:
+
+`wodim -v speed=1 dev=/dev/{{optical_drive}} -audio {{track*.cdaudio}}`
```

Comment:
Is adding the `-v` flag necessary ? Keep in mind that this page should be for beginners so unnecessary options should be kept out.

If it is, please add to the explanation that the command will generate verbose output.